pkg/odo/cli/service/service.go

```diff
@@ -47,8 +46,5 @@ func NewCmdService(name, fullName string) *cobra.Command {
 	appCmd.AddApplicationFlag(serviceDeleteCmd)
 	appCmd.AddApplicationFlag(serviceListCmd)
 
-	completion.RegisterCommandHandler(serviceCreateCmd, completion.ServiceClassCompletionHandler)
-	completion.RegisterCommandHandler(serviceDeleteCmd, completion.ServiceCompletionHandler)
```

Comment:
Why did you remove these?

Reply:
Because they already exist in the respective command handlers, so these were redundant

Reply:
Was wondering about this. Cool!